### DISCUSSION TITLE
Add support for curtain

### DISCUSSION THREAD

**Discussion by sonar98** [andrei-tatar/node-red-contrib-smartnora#98] [Ideas]
Google smart home supports the Curtain device. 
It would be nice if smartnora also supported that.

https://developers.google.com/assistant/smarthome/guides/curtain

**Comment by andrei-tatar** [comment]
@sonar98 curtains are already supported. It's an `open/close` node with `Device Type` set to `Curtain`:
![image](https://user-images.githubusercontent.com/6473183/133560110-1e0b9f34-1b3c-4ff0-9a0a-3d293824a7da.png)
